GPS/client.py

```diff
@@ -0,0 +1,49 @@
+
+""" This Python program represents the client for a simulation of GPS communication.
+
+This sub System communicates with binary strings using SOCK_SEQPACKET and AF_UNIX. 
+This server recieves incoming commands from the client and echoes back the requested
+information.
+
+At the moment, the three commands are:
+    - time          - Request the time and date
+    - latlong       - Request the latitude and longitude information
+    - returnstate   - Request the return state (on/off)
+    - ping          - Ping the server
+    - disconnect    - closes connection to server
+    - terminate     - terminates the server and closes connection
+
+    
+To test the server/client you must run both files in WSL to ensure the use of a 
```

Comment:
You probably don't need to say WSL here, people on linux and mac have a unix environment as well. Maybe just say not windows if you want to specify.